File: resources/template_main.java
package {{PACKAGE}};

import org.bukkit.plugin.java.JavaPlugin;

public class {{CLASS_NAME}} extends JavaPlugin {

    @Override
    public void onEnable() {
        // Plugin startup logic
        getLogger().info("{{CLASS_NAME}} has been enabled!");
    }

    @Override
    public void onDisable() {
        // Plugin shutdown logic
        getLogger().info("{{CLASS_NAME}} ha been disabled!");
    }
}

File: resources/template_main.kt
package {{PACKAGE}}

import org.bukkit.plugin.java.JavaPlugin

class {{CLASS_NAME}} : JavaPlugin() {

    override fun onEnable() {
        // Plugin startup logic
        logger.info("{{CLASS_NAME}} has been enabled!")
    }

    override fun onDisable() {
        // Plugin shutdown logic
        logger.info("{{CLASS_NAME}} ha been disabled!")
    }
    
}